docs/source/contributing/developing.adoc

```diff
@@ -55,7 +58,10 @@ $ export GOPATH=$HOME/work
 $ export PATH=$PATH:$GOPATH/bin
 ----
 
-NOTE: On Windows operating systems, you use the UI or use `setx` to set the environment variables.
+[NOTE]
+====
+On Windows operating systems, you use the UI or use `setx` to set the environment variables.
```

Comment:
Need to add way to set env variable in PowerShell  or use general phrase like
`... you use the UI or use shell specific approach.`

Reply:
@robin-owen Any comment on it?